src/containers/Passphrase.jsx

```diff
@@ -21,9 +22,12 @@ const mapStateToProps = state => ({
 })
 
 const mapDispatchToProps = (dispatch, ownProps) => ({
-  onPassphraseSimpleSubmit: (current, newVal) => {
+  onPassphraseSimpleSubmit: (currentPassphrase, newPassphrase, hint) => {
     return (
-      dispatch(updatePassphrase(current, newVal))
+      dispatch(updateHint(hint))
+        .then(() =>
+          dispatch(updatePassphrase(currentPassphrase, newPassphrase))
+        )
```

Comment:
I think using await would be more readable.

Reply:
See https://github.com/cozy/cozy-settings/pull/269/commits/2f531491dd22f5685ff995fbe3d0a404b3c51633